testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/ReadOnlyModeTestCase.java

```diff
@@ -69,29 +76,37 @@ public void testConfigurationNotUpdated() throws Exception {
 
     @Test
     public void testReadOnlyConfigurationDirectory() throws Exception {
-        // We ignore the test on Windows to prevent in case of errors the pollution of %TMPDIR% with read only directories
-        // On unix machines, the /tmp dir is always deleted on each server boot by root user.
-        Assume.assumeFalse(TestSuiteEnvironment.isWindows());
-
         final Path jbossHome = Paths.get(System.getProperty("jboss.home"));
         final Path configDir = jbossHome.resolve("standalone").resolve("configuration");
         final Path standaloneTmpDir = jbossHome.resolve("standalone").resolve("tmp");
-        final Path osTmpDir =  Paths.get("/tmp");
+        final Path osTmpDir = TestSuiteEnvironment.isWindows() ? new File("target", "tmp").toPath().toAbsolutePath() : Paths.get("/tmp");
+        if(Files.notExists(osTmpDir)) {
+            Files.createDirectories(osTmpDir);
+        }
         final Path roConfigDir = Files.createTempDirectory(osTmpDir, "wildfly-test-suite-");
-
         PathUtil.copyRecursively(configDir, roConfigDir, true);
 
-        Set<PosixFilePermission> perms = new HashSet<>();
-
-        perms.add(PosixFilePermission.OWNER_READ);
-        perms.add(PosixFilePermission.OWNER_EXECUTE);
-        perms.add(PosixFilePermission.GROUP_READ);
-        perms.add(PosixFilePermission.GROUP_EXECUTE);
-        perms.add(PosixFilePermission.OTHERS_READ);
-        perms.add(PosixFilePermission.OTHERS_EXECUTE);
-
-        Files.getFileAttributeView(roConfigDir, PosixFileAttributeView.class).setPermissions(perms);
-
+        if (TestSuiteEnvironment.isWindows()) {
```

Comment:
This seems odd to me, for Windows an ACL is constructed that denies the listed permissions, which seems correct, but on all other OSes are added as allowed. I'm probably missing something here though?

Reply:
@ehsavoie FYI ^^^

Reply:
Well 1st, I only looked at Windows but the problem is that the permissions are quite complex with a lot of principals so I reduced the change to the owner only and removing a permission isn't sufficient in Windows you have to DENY it.